index.html

```diff
@@ -5,11 +5,104 @@
+                <div class="main-content-box1 image-width">
+                    <div class="frontground">
+                        <a class="box-title sm-title no-text-decoration" href="#">iPhone 5 <span>S</span><br><small
+                                class="sm-text">Forward
+                                thinking.</small></a>
+                    </div>
+                </div>
+                <div class="main-content-box2 image-width">
+                    <div class="frontground">
+                        <a class="box-title sm-title no-text-decoration" href="#">iPhone 5 <span>S</span><br><small
+                                class="sm-text">for
+                                the
+                                colorful.</small></a>
+                    </div>
+                </div>
+                <div class="main-content-box3 image-width">
+                    <div class="frontground">
+                        <a class="box-title lg-title no-text-decoration" href="#">Happy Birthday,
+                            Mac.<br><small class="lg-text">Explore
+                                30 Years of
+                                innovation on
+                                the Mac.</small></a>
+                    </div>
+                </div>
+                <div class="main-content-box4 image-width">
+                    <div class="frontground">
+                        <a class="box-title lg-title no-text-decoration" href="#">Supplier
+                            Responsibility<br><small class="lg-text">The
+                                latest results
+                                of
+                                our
+                                worldwide
+                                efforts.</small></a>
+                    </div>
+                </div>
+            </div>
+        </main>
+        <footer>
+            <div class="footer-link">
+                <div class="footer-link-right parent-element">
+                    <a class="no-text-decoration footer-text-size" href="#">Site Map</a>
+                    <a class="no-text-decoration footer-text-size" href="#">Hot News</a>
+                    <a class="no-text-decoration footer-text-size" href="#">RSS Feeds </a>
+                    <a class="no-text-decoration footer-text-size" href="#">Media Info</a>
+                    <a class="no-text-decoration footer-text-size" href="#">Environment</a>
+                    <a class="no-text-decoration footer-text-size" href="#">Job Opportunities</a>
+                    <a class="no-text-decoration footer-text-size" href="#">Contact Us</a>
+                    <button class="child-element" type="button"><img class="us-flag" src="images/icon-usa.png"
+                            alt="usa-flag-icon"></button>
+                </div>
+                <div class="footer-link-left">
+                    <p class="footer-text-size">Shop the <a class="no-text-decoration" href="#">Apple online store</a>
```

Comment:
- [ ] The color of the link should be the same as the original.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/92412045-b8ce5980-f120-11ea-9f64-4e6105eddf0a.png) | ![image](https://user-images.githubusercontent.com/61479648/92412054-c1bf2b00-f120-11ea-9ce3-2b0c41d09298.png) |